### PR TITLE
Fix a big pile of linking errors.

### PR DESCRIPTION
**Pull request by miketaylr**
Though not all. See #162 for the remaining ones.

***
<a href="https://pr-preview.s3.amazonaws.com/miketaylr/background-fetch/pull/163.html" title="Last updated on Oct 19, 2021, 1:18 PM UTC (3bb1699)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/background-fetch/163/5697424...miketaylr:3bb1699.html" title="Last updated on Oct 19, 2021, 1:18 PM UTC (3bb1699)">Diff</a>

### CODE REVIEW

**Comment by miketaylr** [comment]
PTAL @beverloo (twitter tells me jake is on PTO!)

File: index.bs [hunk 1]
```diff
@@ -150,7 +151,7 @@ A <dfn>background fetch</dfn> consists of:
       :: |icon|'s [=image resource/src=].
       : [=request/Client=]
       :: |environment|.
-      : [=request/Keepalive flag=]
+      : [=request/Keepalive|keepalive flag=] flag
       :: Set.
```
Comment (annevk) [inline index.bs:155]:
```suggestion
      : [=request/Keepalive=]
      :: True.
```

File: index.bs [hunk 2]
```diff
@@ -278,7 +279,7 @@ Note: This algorithm manages the fetching of a [=/background fetch record=]. One
 
     Note: At this point the request is entirely held in storage, even if it started as a stream.
 
-  1. Set |request|'s [=request/keepalive flag=].
+  1. Set |request|'s [=request/keepalive|keepalive flag=].
```
Comment (annevk) [inline index.bs:282]:
```suggestion
  1. Set |request|'s [=request/keepalive=] to true.
```

**Comment by miketaylr** [comment]
PTAL @beverloo @jakearchibald, thanks!

**Review by jakearchibald — APPROVED** [review]
Thank you! I'll take a look at the rest

**Pull request merged**